wolfcrypt/src/port/iotsafe/iotsafe.c

```diff
@@ -859,6 +881,14 @@ static int iotsafe_sign_hash(byte *privkey_idx, uint16_t id_size,
             byte sig_hdr[3];
             if (hex_to_bytes(resp, sig_hdr, 3) < 0) {
                ret = BAD_FUNC_ARG;
+#ifdef IOTSAFE_SIGN_NO_PADDING
+            } else if ((sig_hdr[0] == IOTSAFE_TAG_SIGNATURE_FIELD) &&
```

Comment:
Please move brace outside #ifdef and explicitly add one for this else if. Have the braces mixed with #ifdef is problematic.